File: GTTS/GTTS.Web/ClientApp/src/app/models/INL.SearchService.Models/isearch-training-events_item.ts



import { SearchTrainingEventLocations_Item } from './search-training-event-locations_item';
import { SearchTrainingEventKeyActivities_Item } from './search-training-event-key-activities_item';

export interface ISearchTrainingEvents_Item {
  
	TrainingEventID: number;
	Name: string;
	NameInLocalLang: string;
	CountryID: number;
	TrainingUnitID?: number;
	TrainingUnitAcronym: string;
	TrainingUnit: string;
	OrganizerAppUserID?: number;
	OrganizerFullName: string;
	ParticipantCount: number;
	TrainingEventType: string;
	TrainingEventStatusID?: number;
	TrainingEventStatus: string;
	EventStartDate?: Date;
	EventEndDate?: Date;
	Locations?: SearchTrainingEventLocations_Item[];
	KeyActivities?: SearchTrainingEventKeyActivities_Item[];

}

File: GTTS/INL.SearchService/Models/ISearchTrainingEvents_Item.cs
using System;
using System.Collections.Generic;

namespace INL.SearchService.Models
{
    public interface ISearchTrainingEvents_Item
    {
        long TrainingEventID { get; set; }
        string Name { get; set; }
        string NameInLocalLang { get; set; }
        int CountryID { get; set; }
        long? TrainingUnitID { get; set; }
        string TrainingUnitAcronym { get; set; }
        string TrainingUnit { get; set; }
        int? OrganizerAppUserID { get; set; }
        string OrganizerFullName { get; set; }
        int ParticipantCount { get; set; }
        string TrainingEventType { get; set; }
        int? TrainingEventStatusID { get; set; }
        string TrainingEventStatus { get; set; }
        DateTime? EventStartDate { get; set; }
        DateTime? EventEndDate { get; set; }
        List<SearchTrainingEventLocations_Item> Locations { get; set; }
        List<SearchTrainingEventKeyActivities_Item> KeyActivities { get; set; }
    }
}
